hack/build-manifests.sh

```diff
@@ -33,7 +33,7 @@ warning_message="# WARNING: This is an auto generated file, do not modify this f
 main() {
     cd "$SCRIPT_DIR/.."
     local ret=0
-    find task/*/*/*.yaml -maxdepth 0 | awk -F '/' '{ print $0, $2, $3, $4 }' | \
+    find task -maxdepth 3 -type f -name "*.yaml" | awk -F '/' '{ print $0, $2, $3, $4 }' | \
```

Comment:
To make sure it's equivalent to the previous behavior, maybe `-mindepth` should also be 3? I don't _think_ we have any yaml file at levels 1 or 2, but I'm not sure

Reply:
> isn't it actually better to either fail

Good point. I'm not sure if the current behavior would result in a sensible failure though, given that the `awk` on the same line expects paths with exactly 4 components. I suspect we'd just end up skipping the files anyway, but in a more obfuscated way